src/AltinnCore/Designer/Controllers/RepositoryController.cs

```diff
@@ -60,10 +60,10 @@ public List<Repository> Search(RepositorySearch repositorySearch)
         }
 
         /// <summary>
-        /// Returns a given service repository
+        /// Returns a given app repository
         /// </summary>
         /// <param name="owner">The service owner</param>
```

Comment:
the app owner / org

Reply:
I do not quite follow this one. This method will return the repository object, which is at the "[repos]/[org]/[app]" location in Gitea. ($"/repos/{owner}/{repository}", where "repository" is practically the same as "app").

Reply:
Im refering to the input parameter owner.. But I guess you will be fixing that in the next round. it should be org. Line 65.

Reply:
Yes, correct.